### PR TITLE
feature/1093 Alter DB schema for settlements

### PR DESCRIPTION
**Pull request by ggrg**
- Changes to the OSS-Settlement-BRD
- Updated DB schema diagram
- Added ERD & DDL files

### CODE REVIEW

File: mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md [hunk 1]
```diff
@@ -1153,51 +1091,37 @@ should be changed to check that some of the settlement windows associated with
...
-settlement request rejected. [Story \#14]
+settlement request rejected. [Story \#1096]
 
 #### Updating the progress of a settlement.
 
 The process to update the status of a settlement described in Section 5.1.7.1.2
 above allows the caller to specify the currency whose status is to be changed.
 Since the settlement will now be for a settlement model, this parameter should
-be removed. [Story \#15]
+be removed. (@ggrg: not sure what needs to be removed)
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md]: Why the notes?

File: mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md [hunk 2]
```diff
@@ -1153,51 +1091,37 @@ should be changed to check that some of the settlement windows associated with
...
 
 When the status for a settlement participant is updated, this will result in
 changes to the status of all the records in the **settlementContentAggregation**
-table for the given participant. These records can be identified using the
+table for the given participant (@ggrg: we don't have attached state to this table). These records can be identified using the
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md]: Why the notes?

File: mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md (hunk 1 above)
Comment (ggrg) [reply mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md]: Thanks, for spotting it. It's been resolved.

File: mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md (hunk 2 above)
Comment (ggrg) [reply mojaloop-technical-overview/central-settlements/oss-settlement-fsd.md]: Thanks, for spotting it. It's been resolved.

**Pull request merged**